app/lib/wolf_waagen_api/import.rb

```diff
@@ -98,17 +99,47 @@ def start_date
 
     def save_series_points(series:)
       sensors = api_sensors_by_series_id(series_id: series.id)
+      interval = accumulation_interval_by_series_id(series_id: series.id)
 
       sensors.each do |sensor|
         series.points.each do |point|
+          value = accumulated_value(sensor: sensor, interval: interval, point: point)
+
           Sensor::Reading.find_or_create_by(
             sensor: sensor,
             created_at: point.datetime,
-            calibrated_value: point.data,
-            uncalibrated_value: point.data
+            calibrated_value: value,
+            uncalibrated_value: value
           )
         end
       end
     end
+
+    def accumulation_interval_by_series_id(series_id:)
+      type = SENSOR_TYPES.find { |t| t[:series_id] == series_id }
+      type[:accumulate] if type&.key?(:accumulate)
```

Comment:
```suggestion
      type[:accumulate]
```
If `type[:accumulate]` is nil, that's fine, no?